package.json

```
@@ -84,5 +84,9 @@
     "stylelint-config-recommended": "^6.0.0",
     "stylelint-config-sass-guidelines": "^9.0.1",
     "stylelint-no-unsupported-browser-features": "^5.0.2"
+  },
+  "dependencies": {
+    "react-slick": "^0.28.1",
```

do we need both packages? I thought that `react-slick` was a port of `slick-carousel`. So that all of the styles should also be in react-slick

---

Yes.  One is the JS, the other CSS

---

@JackHowa : Check out some notes I added in the PR under Description, Dependencies. `react-slick` is the React code, but doesn't have the `.scss` version of the styles of `slick-carousel` (the project it bloomed out of). It's unfortunate to bring in `slick-carousel` just for the Sass files, but that's the current reasoning.

In React-Slick's repo, there's `slick.css` and `slick-theme.css`:
https://github.com/akiran/react-slick/tree/master/docs

Unfortunately I think we want, need the Sass files that live here:
https://github.com/kenwheeler/slick/tree/master/slick

cc: @visionbegin

---

wait, I'm confused. Maybe I'm missing something around this. So, we've effectively "forked" the scss by copying and pasting the scss in? If the intention is to "freeze" the version, we should reflect that in the `package.json`
```suggestion
    "react-slick": "0.28.1",
```

> If we are against linking to the foundational Sass, then we can copy that in, but we then should freeze the version of the react-slick component we are currently using.

---

has this been addressed @visionbegin @ecarlisle ? I'd recommend freezing the version via specifying the package number explicitly

---

If we lock slick, we should also lock slick-carousel.  If not, they should at least have the same upgrade markers (~, ^, etc.) in similar locations.

---

@JackHowa We agreed yesterday that we wouldn't freese `react-slick`.  Instead, we would import the base scss file from `slick-carousel` but copy in the theme file from that package so that we can integrate it into the advanced styling.